code/obj/item/device/radios/intercoms.dm

```diff
@@ -7,14 +7,31 @@
 #endif
 	anchored = 1.0
 	plane = PLANE_NOSHADOW_ABOVE
-	mats = 0
+	mats = 3
+	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_WIRECUTTERS | DECON_MULTITOOL
 	chat_class = RADIOCL_INTERCOM
 	var/number = 0
 	rand_pos = 0
 	desc = "A wall-mounted radio intercom, used to communicate with the specified frequency. Usually turned off except during emergencies."
 	hardened = 0
 
+/obj/item/device/radio/intercom/proc/change_dir(obj/item/AM, old_dir, new_dir)
+	src.pixel_x = 0
+	src.pixel_y = 0
+	switch(new_dir)
+		if(NORTH)
+			src.pixel_y = -21
+		if(SOUTH)
+			src.pixel_y = 24
+		if(EAST)
+			src.pixel_x = -21
+		if(WEST)
+			src.pixel_x = 21
+
 /obj/item/device/radio/intercom/New()
+	//Register before calling our parent
+	//There's a possible race condition in assembly from a frame if you don't
```

Comment:
Seems interesting. Could you expand on this a bit (here, not as a comment in code)?

Reply:
Just concern about a lag spike causing it to be left in the original direction but eh, removed